_install-and-configure/configuring-opensearch/configuration-system.md

```diff
@@ -11,14 +11,20 @@ For an overview of creating an OpenSearch cluster and examples of configuration
 
 OpenSearch supports the following system settings:
 
-- `cluster.name` (Static, string): The cluster name. 
+- `cluster.name` (Static, string): The cluster name. Default is `opensearch`.
 
-- `node.name` (Static, string): A descriptive name for the node.
+- `node.name` (Static, string): A descriptive name for the node. Required.
 
-- `node.roles` (Static, list): Defines one or more roles for an OpenSearch node. Valid values are `cluster_manager`, `data`, `ingest`, `search`, `ml`, `remote_cluster_client`, and `coordinating_only`. 
+- `node.roles` (Static, list): Defines one or more roles for an OpenSearch node. Valid values are `cluster_manager`, `data`, `ingest`, `search`, `ml`, `remote_cluster_client`, and `coordinating_only`. Default is `cluster_manager,data,ingest,remote_cluster_client`.
 
-- `path.data` (Static, string): A path to the directory where your data is stored. Separate multiple locations with commas. 
+- `path.data` (Static, string): A path to the directory where your data is stored. Separate multiple locations with commas. The default depends on the deployment method, see following examples of default `path.data` per deployment method:
+     - RPM/DEB: `/var/lib/opensearch`
+     - Docker: `/usr/share/opensearch/data`
+     - Tarball: `$OPENSEARCH_HOME/data`
```

Comment:
Technically everything is $OPENSEARCH_HOME/data.
The DEB/RPM one was using a symlink from $OPENSEARCH_HOME/data to /var/lib/opensearch.
Same as docker where $OPENSEARCH_HOME is same as on DEB/RPM which is /usr/share/opensearch.

Without specifying path.data it is always $OPENSEARCH_HOME/data to start with.